streams/src/main/java/org/apache/kafka/streams/state/internals/InMemoryTimeOrderedKeyValueBuffer.java

```diff
@@ -61,6 +62,7 @@
         new RecordHeaders(new Header[] {new RecordHeader("v", new byte[] {(byte) 1})});
     private static final RecordHeaders V_2_CHANGELOG_HEADERS =
         new RecordHeaders(new Header[] {new RecordHeader("v", new byte[] {(byte) 2})});
+    private static final String METRIC_SCOPE = "in-memory-suppression";
```

Comment:
I had to add this metric scope (a.k.a. store type) for the metrics tags. Now the tags would 
```
type=stream-state-metrics,
thread-id=[threadId],
task-id=[taskId],
in-memory-suppression-state-id=[storeName]
```

Reply:
What's the tags look like before?

Reply:
Before it was:
```
type=stream-buffer-metrics,
client-id=[threadId]
task-id=[taskId]
buffer-id=[storeName]
```